markdown/ch2.markdown

```diff
@@ -105,6 +97,14 @@ We will use a Gaussian kernel to smooth out bumps in our data so we can get a
 better idea of the overall shape.
 
 ```python
+# Make all plots appear inline in the Jupyter notebook
+%matplotlib inline
+# Set custom plotting style
+import matplotlib.pyplot as plt
+plt.style.use('style/elegant.mplstyle')
```

Comment:
I decided, not we 😛 
I felt like it was slightly less awkward, since a few other chapters had already done this and it seemed to work. And I didn't want to put it before the motivating code snippets, since I wanted them to be the first code in each chapter.